mne/io/nirx/tests/test_nirx.py

```diff
@@ -13,8 +13,8 @@
 from mne.io.tests.test_raw import _test_raw_reader
 from mne.transforms import apply_trans, _get_trans
 from mne.utils import run_tests_if_main
-from mne.preprocessing._beer_lambert_law import _probe_distances,\
-    short_channels
+from mne.preprocessing.nirs import nirs_source_detector_distances,\
+    nirs_short_channels
```

Comment:
if you put functions in a nirs submodule no need to replicate nirs in the function name.

you could do

```
from mne.preprocessing import nirs
nirs.source_detector_distances(...)
```